apps/ideas/templates/advocate_europe_ideas/idea_detail.html

```diff
@@ -61,7 +64,7 @@
         </div>
         <div class="container-medium ideadetail-comments">
             <h4>{% trans "Write comment" %}</h4>
-            {% react_comments ideasketch %}
+            {% react_comments idea %}
```

Comment:
Is `idea` always the BaseClass in this context or could it also be the proposal?